io.itch.Itch.yaml

```diff
@@ -0,0 +1,143 @@
+    no-autodownload: true
+    version: "22.08"
+
+  com.valvesoftware.Steam.Utility.gamescope:
+    add-ld-path: lib
+    directory: utils/gamescope
+
+modules:
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+        commit: 55a60d110301e8dd37b631503c3524ab7baaf7aa
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+          
+  - name: itch-resources
+    buildsystem: simple
+    build-commands:
+      - install -D -t "${FLATPAK_DEST}/share/applications/" ./io.itch.Itch.desktop
+      - mkdir -p "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/"
+      - install -D -t "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/" ./io.itch.Itch.svg
+      - install -D -t "${FLATPAK_DEST}/share/metainfo/" ./io.itch.Itch.metainfo.xml
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/itchio/press-kit/master/logos/app-icon.svg
+        dest-filename: io.itch.Itch.svg
+        sha256: eaf979c6c7c315a7004971bc82bbced24c06d7d4fff7f518d7cfd83863ea4285
+        
+      - type: file
+        path: io.itch.Itch.desktop
+        
+      - type: file
+        path: io.itch.Itch.metainfo.xml
+
+  - name: itch
+    buildsystem: simple
+    build-commands:
+      - mv * /app/bin/
+    sources:
+      - type: script
+        dest-filename: itch-run
+        commands:
+          - zypak-wrapper.sh /app/bin/itch "$@"
+        
+      - type: archive
+        strip-components: 0
+        url: https://broth.itch.ovh/itch/linux-amd64/25.5.1/itch-linux-amd64.zip
+        sha256: fdd916c79beb90c19c7ee404e11398e5f2f5c7c402ae68910b4f70e6918071ed
+#        x-checker-data:
+#          type: rotating-url
+#          url: https://broth.itch.ovh/itch/linux-amd64/25.5.1/itch-linux-amd64.zip
+#          pattern: https://broth.itch.ovh/itch/linux-amd64/v([0-9.]+)/itch-linux-amd64.zip
+
+  - name: firejail
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/netblue30/firejail.git
+        tag: "0.9.70"
+        commit: b4b08d21cd95725c9d55dfdb6987fcc6d7893247
+#        x-checker-data:
+#          type: git
+#          tag-pattern: ^v([\\d.]+)$
+
+  - name: gamescope
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/utils/gamescope
+
+  - name: gamemode
+    buildsystem: meson
+    config-opts:
+#      - -Dwith-systemd=false
+#      - -Dwith-daemon=false
+      - -Dwith-examples=false
+      - -Dwith-util=false
+      - -Dwith-sd-bus-provider=no-daemon
+    sources: &gamemode_sources
+      - type: git
+        url: https://github.com/FeralInteractive/gamemode.git
+        tag: "1.7"
+        commit: 4dc99dff76218718763a6b07fc1900fa6d1dafd9
+#        x-checker-data:
+#          type: git
+#          tag-pattern: ^v([\\d.]+)$
+
+  - name: gamemoderun
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 data/gamemoderun -t /app/bin
+    sources: *gamemode_sources
```

Comment:
```suggestion
  - name: gamemode
    buildsystem: meson
    config-opts:
      - -Dwith-examples=false
      - -Dwith-util=false
      - -Dwith-sd-bus-provider=no-daemon
    post-install:
      - install -Dm755 data/gamemoderun -t /app/bin
    sources:
      - type: git
        url: https://github.com/FeralInteractive/gamemode.git
        tag: "1.7"
        commit: 4dc99dff76218718763a6b07fc1900fa6d1dafd9
#        x-checker-data:
#          type: git
#          tag-pattern: ^v([\\d.]+)$
```

Reply:
Intriguing, I shall remember this for the future.